sdc/datatypes/hpat_pandas_dataframe_functions.py

```diff
@@ -1532,7 +1536,7 @@ def df_getitem_bool_series_idx_main_codelines(self, idx):
                       f'  self_index = self.index',
                       f'  idx_reindexed = sdc_reindex_series(idx._data, idx.index, idx._name, self_index)',
                       f'  res_index = getitem_by_mask(self_index, idx_reindexed._data)',
-                      f'  selected_pos = getitem_by_mask(numpy.arange(length), idx_reindexed._data)']
+                      f'  selected_pos = getitem_by_mask(range(0, length), idx_reindexed._data)']
```

Comment:
Looks like it is equal to:
```suggestion
                      f'  selected_pos = getitem_by_mask(range(length), idx_reindexed._data)']
```